packages/block-editor/src/components/inner-blocks/use-inner-block-template-sync.js

```diff
@@ -17,9 +17,9 @@ import { synchronizeBlocksWithTemplate } from '@wordpress/blocks';
  * is meant to be locked (e.g. templateLock = "all"), then we replace the inner
  * blocks with the correct value after synchronizing it with the template.
  *
- * @param {string} clientId     The block client ID.
- * @param {Object} template     The template to match.
- * @param {string} templateLock The template lock state for the inner blocks. For
+ * @param {string}  clientId                       The block client ID.
+ * @param {Object}  template                       The template to match.
+ * @param {string}  templateLock                   The template lock state for the inner blocks. For
```

Comment:
This needs updating and the lines below to line 31, looks like autofix missed the `@param` on line 28